Modules/Scripted/EditorLib/EditColor.py

```diff
@@ -50,27 +51,85 @@ def cleanup(self, QObject=None):
   def create(self):
```

Comment:
Could you consider moving the ColorBox and new terminology into a python widget like it is done for `LabelStructureListWidget` ? . See https://github.com/Slicer/Slicer/blob/master/Modules/Scripted/EditorLib/LabelStructureListWidget.py